docs/index.md

```diff
@@ -5,20 +5,20 @@ styles: example
 scripts: example
 ---
 
-## Palette Picker. A customizable colour template data type for Umbraco.
+## Palette Picker. A customizable colour palette picker for Umbraco.
 
-Palette Picker allows editors to select a predefined colour template for any particular page of their site.
+Palette Picker allows an editor to choose a colour palette to use on a page from a predefined selection of colour palettes.
 
-This can be especially useful for document types where we want colours to change often or be customizable.
+This can be especially useful for pages where you would like colours to change often or be easily customizable.
 
-It makes it really simple for editors to change the colour template value, while it requests only a minimum configuration from developers to set up the available colour templates. It will generate a series of CSS rules that can imported into any view, and then use the specific classes accordingly on any colour-custom element of the view. The CSS rules included contain the properties color, background-color and border-color for main classes, as well as for other optional pseudo-elements and pseudo-classes. For more info, please refer to the [Documentation]({{ site.baseurl }}/documentation) page.
+It makes it really simple for editors to change the colour template value, while it requires only minimum configuration from developers to set up the available colour palettes. Palette Picker will generate a series of CSS rules that can included on any page, these can be used to style elements across the page with the same color theme. The CSS rules included contain the properties color, background-color and border-color for main classes, as well as for other optional pseudo-elements and pseudo-classes. For more info, please refer to the [Documentation]({{ site.baseurl }}/documentation) page.
```

Comment:
Should it be "colour palette value" too?.
Missing word on "a series of CSS rules that can (be) included on any page.